app/controllers/orders_controller.rb

```diff
@@ -1,2 +1,42 @@
 class OrdersController < ApplicationController
```

Comment:
Please ignore this file for now! I decided to work on this in a separate branch.